driver-testsuite/tests/Basic/CookieTest.php

```diff
@@ -45,6 +45,10 @@ public function testCookie()
         $this->getSession()->setCookie('srvr_cookie', null);
         $this->getSession()->reload();
         $this->assertContains('Previous cookie: NO', $this->getSession()->getPage()->getText());
+
+        $this->getSession()->setCookie('semicolon', 'foo;bar;baz');
+        $this->getSession()->visit($this->pathTo('/cookie_page4.php'));
```

Comment:
You mean https://github.com/minkphp/Mink/blob/master/driver-testsuite/web-fixtures/cookie_page2.php page? There is no line 14 there.

Yes, you can name your cookie `srvr_cookie` and use that page too.